### PR TITLE
doc(deploy): describe policy override and defaults

### PR DESCRIPTION
**Pull request by uguy**
https://bonitasoft-bonita-continuous-delivery-doc-build_preview-pr-191.surge.sh/bcd/3.6/livingapp_deploy#_deployment_descriptor_file

### CODE REVIEW

File: modules/ROOT/pages/livingapp_deploy.adoc [hunk 1]
```diff
@@ -73,6 +73,13 @@ Each artifact to deploy must be defined with the following attributes:
 * `file`: (Mandatory) the relative path to the artifact in the Application Archive
 * `policy`: (Optional) the name of the policy to apply in case the same artifact is already present in the target Bonita platform. If omitted, then the <<supported-policies,default policy>> of the artifact's type will be applied.
 
+[NOTE]
+====
+- If a deploy policy is declared in a `deploy.json` file, it overrides the policy specified as a CLI options (like `--development-mode`).
+- If no policy is specified in the `deploy.json` file, the policy specified as a CLI options will be applied.
```
Comment (JeremJR) [inline modules/ROOT/pages/livingapp_deploy.adoc]:
```suggestion
- If a deploy policy is declared in a `deploy.json` file, it overrides the policy specified as a CLI option (like `--development-mode`).
- If no policy is specified in the `deploy.json` file, the policy specified as a CLI option will be applied.
```

File: modules/ROOT/pages/livingapp_deploy.adoc [hunk 2]
```diff
@@ -73,6 +73,13 @@ Each artifact to deploy must be defined with the following attributes:
 * `file`: (Mandatory) the relative path to the artifact in the Application Archive
 * `policy`: (Optional) the name of the policy to apply in case the same artifact is already present in the target Bonita platform. If omitted, then the <<supported-policies,default policy>> of the artifact's type will be applied.
 
+[NOTE]
+====
+- If a deploy policy is declared in a `deploy.json` file, it overrides the policy specified as a CLI options (like `--development-mode`).
```
Comment (JeremJR) [inline modules/ROOT/pages/livingapp_deploy.adoc]: You say "like `--development-mode`" but is there other ways to specify policies with a CLI option?

**Review by JeremJR — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/livingapp_deploy.adoc [hunk 3]
```diff
@@ -170,6 +177,22 @@ Each artifact to deploy must be defined with the following attributes:
...
+- Process: `IGNORE_DUPLICATES`
+- Profile: `IGNORE_IF_ANY_EXISTS`
+- Application: `REPLACE_DUPLICATES`
+
+==== Development mode
+
+- Organization: `MERGE_DUPLICATES`
+- Process: `REPLACE_DUPLICATES`
+- Profile: `REPLACE_DUPLICATES`
+- Application: `REPLACE_DUPLICATES`
```
Comment (JeremJR) [inline modules/ROOT/pages/livingapp_deploy.adoc:194]: There is not policy for Pages, Layouts, Themes, BusinessDataModel and BdmAccessControl too?

Comment (uguy) [reply modules/ROOT/pages/livingapp_deploy.adoc:194]: Nothing like this found in the code for them.

**Pull request merged**